runblender.ipynb

```diff
@@ -17,107 +17,92 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "MNd7jnnSkuUD",
+        "id": "vdWkwAMYN52S",
         "colab_type": "text"
       },
       "source": [
-        "#Setup\n",
-        "Specify your desired blender version and the path to your blend file within google drive.\n",
-        "\n",
-        "###Info\n",
-        "If you do need more information on parameters etc. look here: [Blender CLI Wiki](https://docs.blender.org/manual/en/latest/advanced/command_line/arguments.html)"
+        "Google Drive Mount"
       ]
     },
     {
       "cell_type": "code",
       "metadata": {
-        "id": "8dFNjYGTgNjR",
-        "colab_type": "code",
-        "colab": {}
-      },
-      "source": [
-        "#@title Setup\n",
-        "#@markdown Please configure your setup\n",
-        "\n",
-        "blender_version = 'blender2.83' #@param [\"blender2.80\", \"blender2.81\", \"blender2.82\", \"blender2.83\"] {allow-input: false}\n",
-        "path_to_blend = 'path/to/blend.blend'  #@param {type: \"string\"}\n",
-        "output_path = 'output/dir/in/drive/output####.png'  #@param {type: \"string\"}\n",
-        "gpu_enabled = True #@param {type:\"boolean\"}\n",
-        "cpu_enabled = False #@param {type:\"boolean\"}\n",
-        "#@markdown ---\n"
-      ],
-      "execution_count": 0,
-      "outputs": []
-    },
-    {
-      "cell_type": "code",
-      "metadata": {
-        "id": "GPJ9fmoB6PWM",
+        "id": "QRzNmiHN8Xr0",
         "colab_type": "code",
-        "colab": {}
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 129
+        },
+        "outputId": "31e4153e-4d3b-4ef9-d122-f793a10a3afc"
       },
       "source": [
-        "if blender_version == \"blender2.80\":\n",
-        "    download_path=\"https://ftp.halifax.rwth-aachen.de/blender/release/Blender2.80/blender-2.80-linux-glibc217-x86_64.tar.bz2\"\n",
-        "elif blender_version == \"blender2.81\":\n",
-        "    download_path=\"https://ftp.halifax.rwth-aachen.de/blender/release/Blender2.81/blender-2.81-linux-glibc217-x86_64.tar.bz2\"\n",
-        "elif blender_version == \"blender2.82\":\n",
-        "    download_path=\"https://ftp.halifax.rwth-aachen.de/blender/release/Blender2.82/blender-2.82-linux64.tar.xz\"\n",
-        "elif blender_version == \"blender2.83\":\n",
-        "    download_path=\"https://ftp.halifax.rwth-aachen.de/blender/release/Blender2.83/blender-2.83.0-linux64.tar.xz\""
+        "from google.colab import drive\n",
+        "drive.mount('/gdrive')"
       ],
-      "execution_count": 0,
-      "outputs": []
+      "execution_count": 1,
+      "outputs": [
```

Comment:
Would you mind replacing the outputs with empty arrays?

Reply:
Ok I will do so